File: C#/백준/Silver/17509. And the Winner Is．．． Ourselves！/And the Winner Is．．． Ourselves！.cs
using System;

namespace q{
    class w{
        static void Main(){
            int p = 0;
            int time = 0;
            int[] aa = new int[11];
            for (int i=0;i<11;i++){
                int[] bb = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                aa[i] = bb[0];
                p+=bb[1]*20;
            }
            Array.Sort(aa);
            for (int i=0;i<11;i++){
                time += aa[i];
                p+=time;
            }
            Console.WriteLine(p);
        }
    }
}

File: Python/백준/Silver/17509. And the Winner Is．．． Ourselves！/And the Winner Is．．． Ourselves！.py
p = 0
aa=[]
for i in range(11):
    aa.append(list(map(int,input().split())))
aa.sort(key=lambda x:x[0])
time = 0
for i in range(11):
    time+=aa[i][0]
    p+=time+aa[i][1]*20
print(p)